changelog/unicode_table_generator.dd

```diff
@@ -0,0 +1,4 @@
+Unicode table generator is now in Phobos, tables are updated to version 15.
+
+It is likely that this change will result in breakage in code and program usage.
+This is due to a number of factors, the tables being updated so significantly and the table generator not having all its changes commited throughout the years.
```

Comment:
Does your editor support adding newlines to the end of files? :-)

Reply:
You know, it's the oddest thing, Nodepad++ has literally every other feature but just not that one!